docs/visual-testing/mobile-native-testing.md

```diff
@@ -25,6 +25,11 @@ When writing a visual test for mobile apps, we recommend the following:
 ## Limitations
 
 The following features are not yet available for mobile app testing:
-- Full page screenshots
-- DOM capture and inspection
-- [Selective diffing](./selective-diffing.md)
+- [Selective diffing](./selective-diffing.md) is constrained by what is available in Appium's page source.
```

Comment:
What does it mean ? Users may not know exactly what this means (and actually I'm not sure of what this means either).